File: KarateKlub/KarateKlub.Api/Models/ClanoviKubaPregledClanovaVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{
    public class ClanoviKubaPregledClanovaVM
    {
        public class Row {
            public int id;
            public string ime;
            public string prezime;
            public string imeRoditelja;
            public string spol;
            public DateTime datumRodjenja;
            public string mjestoRodjenja;
            public byte[] slika;
            public string kontaktTelefon;
        }

        public List<Row> rows;
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_trener/model/ClanoviKubaPregledClanovaVM.java
package com.example.x.karateklub.area_trener.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class ClanoviKubaPregledClanovaVM implements Serializable {
    public static class Row implements Serializable {
        public int id;
        public String ime;
        public String prezime;
        public String imeRoditelja;
        public String spol;
        public Date datumRodjenja;
        public String mjestoRodjenja;
        public String slika;
        public String kontaktTelefon;

    }

    public List<Row> rows;
}
